### PR TITLE
Many fixups for the docker page

### PR DESCRIPTION
**Pull request by stefwalter**
* Don't make the Memory/CPU usage overflow container details
 * Fix layout of container terminal/logs on details page
 * Don't automatically focus the container terminal
 * Hide certain columns in containers on narrow screens

### CODE REVIEW

**Comment by stefwalter** [comment]
This actually needs a test for the 'Change resource limits' dialog.

**Comment by stefwalter** [comment]
I get this when unselecting the memory usage button:

```
sh: line 0: echo: write error: Invalid argument
```

**Comment by stefwalter** [comment]
Moved the commits for the resource dialog, moving to another pull request.

File: pkg/docker/docker.css [hunk 1]
```diff
@@ -187,16 +182,24 @@
...
 
-.image-col-tags {
-    width: 40%;
-}
-
-.image-col-created {
-    width: 30%;
+.image-col-created,
+.image-col-size-text {
+    display: none !important;
```
Comment (mvollmer) [inline pkg/docker/docker.css]: Likewise, the `!important` is too strong for the conditional media rules below.

File: pkg/docker/docker.css [hunk 2]
```diff
@@ -75,30 +75,32 @@
...
-.container-col-image {
-    width: 20%;
-}
-
-.container-col-command {
-    width: 15%;
+.container-col-cpu,
+.container-col-memory-graph,
+.container-col-memory-text {
+    display: none !important;
```
Comment (mvollmer) [inline pkg/docker/docker.css]: Patternfly has this selector: `table td[class*="col-"]` which accidentally matches our class names.  Grrr.

**Comment by mvollmer** [comment]
Looks good except for the fact that some columns are now always hidden.  This is probably fixable by using class names that don't contain "col-".

File: pkg/docker/docker.css (hunk 2 above)
Comment (stefwalter) [inline pkg/docker/docker.css]: Should we just do ```display: table-cell !important```

**Comment by mvollmer** [comment]
"Hide certain columns in containers on narrow screens" is still open.

**Pull request closed without merging**